apps/medusa-be/.env.template

```
@@ -51,6 +52,21 @@ SUPERADMIN_EMAIL=
 SUPERADMIN_PASSWORD=
 INITIAL_PUBLISHABLE_KEY_NAME="Storefront Publishable Key"
 
+# Product review requests and queued workflow processing
+PRODUCT_REVIEW_REQUEST_MESSAGE="Napiš recenzi produktu"
+PRODUCT_REVIEW_REQUEST_DELAY_MINUTES=10080
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PRODUCT_REVIEW_REQUEST_DELAY_MINUTES key should go before the PRODUCT_REVIEW_REQUEST_MESSAGE key

(UnorderedKey)